docs/guides/argo-cd-crossplane.md

```diff
@@ -0,0 +1,62 @@
+---  
+title: Argo CD
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Configuring Crossplane with Argo CD
+ 
+## Overview
+ 
+[Argo CD](https://argoproj.github.io/cd/) and [Crossplane](https://crossplane.io)
+are a great combination. Argo CD provides GitOps while Crossplane turns any Kubernetes
+cluster into a Universal Control Plane for all of your resources. There are
+configuration details required in order for the two to work together properly.
+This doc will help you understand these requirements.
+ 
+Argo CD synchronizes Kubernetes resource manifests stored in a Git repository
+with those running in a Kubernetes cluster (GitOps). It places a label on
+the running resources to know that it should be tracking them. When it sees
+that label on a running resource, it expects it to match the manifest in the
+repo. Additionally, Argo CD makes use of the Kubernetes Owner Ref field to
+correlate additional resources created by operators as a result of the top-level
+synced resources. Owner Ref enforces a rule that a namespace scoped resource
+cannot be the owner of a cluster scoped resource or a resource in another
+namespace. Because of this, Argo CD cannot correlate relationships that are
+structured in either of those two manners.
+ 
+When Crossplane creates namespace scoped Composite Resource Claims (XRCs), they
+result in cluster scoped Composite Resources (XRs) being created. This is a
+relationship structure that Argo CD cannot correlate. One of the nice features
+in Argo CD is its UI where we can visualize those owner ref relationships and
+gleen information on their state. Because of the inability for a owner ref to be
+defined between XRC and XR, we will not see this visualization for Crossplane
+in the Argo CD UI. If we do not use XRC, and start with XR, we will see the
+visualization. This is a tradeoff that needs to be considered when using the
+two together.

```

Comment:
Somewhere in this guide we should link to https://github.com/crossplane/crossplane/issues/2773 and clarify that this is not "working as intended" - i.e. that we desire a better experience than this.